FantasyUniverse/Scenes/Dashboard/DashboardView.swift

```diff
@@ -9,37 +9,26 @@ import SwiftUI
 
 struct DashboardView: View {
     
-    @AppStorage("email") var username: String = "Anonymous"
-    @State var show = false
-    @State var correct = 0
-    @State var wrong = 0
-    @State var answered = 0
-    @State var selectedSet = ""
     let sets = QuestionSet.allSets
-    
+    @State var showSelectedQuiz = false
+    @State var showListItems = false
+    @State var animationDelay = 0.2
+    @State var selectedSet: QuestionSet = .none
```

Comment:
It would be probably more consistant with the mvvm used on other scenes, but it would have no functionality, only one constant and three variables, so maybe no need for viewModel.